src/style.css

```diff
@@ -0,0 +1,144 @@
+:root {
+  --dark-blue: #060b35;
+  --lghit-grey: #616070;
+  --lghit-blue: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  margin: 40px;
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin: 32px 19px 40px;
+  background-image: url(images/imac.jpeg);
+  background-size: 160px 134px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__info {
+  display: flex;
+  flex-direction: column;
+  margin: 0 16px 16px;
+  width: 166px;
+  height: 54px;
+}
+
+.card__item-model {
+  margin: 0 0 6px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--dark-blue);
+}
+
+.card__code {
+  margin: 0;
+  font-size: 10px;
+  line-height: 10px;
+  color: var(--lghit-grey);
+}
+
+.stars {
+  margin: 0 16px 24px;
+  width: 166px;
+  height: 16px;
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-size: 13px 12px;
+  background-position: left;
+  width: 16px;
+  height: 16px;
+  padding-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars__text {
+  padding-left: 18px;
+  align-self: center;
+  margin: 0;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--dark-blue);
+}
+
+.card__price-block {
+  margin: 0 16px 16px;
+  width: 166px;
+  height: 18px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price {
+  width: 31px;
+  height: 18px;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--lghit-grey);
+}
+
+.card__price-number {
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+  width: 50px;
+  height: 18px;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--dark-blue);
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  margin: 0 16px 16px;
+  width: 164px;
+  height: 38px;
+  border-radius: 5px;
+  background-color: var(--lghit-blue);
+  border: 1px solid var(--lghit-blue);
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.card__button:hover {
+  background-color: ;
```

Comment:
There's an empty value for the `background-color` property. Please set the appropriate value for it 🎨